packages/trader/src/AppV2/Components/TradeParameters/trade-parameters-container.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Localize } from '@deriv/translations';
+import { CSSTransition } from 'react-transition-group';
+import { Text } from '@deriv-com/quill-ui';
+import Guide from '../Guide';
+
+type TTradeParametersContainer = {
+    is_minimized?: boolean;
+    is_minimized_visible?: boolean;
+};
+
+const TradeParametersContainer = ({
+    children,
+    is_minimized,
+    is_minimized_visible,
+}: React.PropsWithChildren<TTradeParametersContainer>) => (
+    <React.Fragment>
```

Comment:
I think we can remove the React. Fragment as it'll always have single parent

Reply:
I'm not sure that we can remove React.Fragment here as there is a ternary inside
<img width="850" alt="Screenshot 2024-07-23 at 8 39 19 AM" src="https://github.com/user-attachments/assets/8bc8f924-f3f3-404f-9f49-21599bd0d2ce">

Reply:
Can we remove the curly brackets before `is_minimized ?  `

Reply:
Yep, sorry) Will remove)

Reply:
Let me push the changes, thank you